com.valvesoftware.Steam.yml

```diff
@@ -177,16 +177,16 @@ modules:
     build-commands:
       - |
         set -e
-        sed -i s:Exec=steam:Exec=/app/bin/steam-wrapper: steam.desktop
-        sed -i s:/usr/bin/steam:/app/bin/steam-wrapper: steam.desktop
-        desktop-file-edit --set-key=StartupWMClass --set-value=Steam steam.desktop
-        desktop-file-edit --remove-key=PrefersNonDefaultGPU steam.desktop
+        sed -i s:/usr/bin/steam:/app/bin/steam: steam.desktop
         desktop-file-edit --remove-key=X-KDE-RunOnDiscreteGpu steam.desktop
+        desktop-file-edit --set-key=StartupWMClass --set-value=Steam steam.desktop
         sed -i s:/usr/lib/:/app/lib/: steam
         PREFIX=/app make install
         ln -sf /bin/true /app/bin/steamdeps
         cp /usr/bin/cmp /app/bin
         install -Dm644 -t /app/share/metainfo com.valvesoftware.Steam.metainfo.xml
+        ln -sf /app/bin/steam-wrapper /app/bin/steam
+        test -x 
```

Comment:
A leftover?

Reply:
This is actually missing the actual path. Will fix.